scout/constants/variant_tags.py

```diff
@@ -49,6 +49,79 @@
 
 VARIANT_CALL = ("Pass", "Filtered", "Not Found", "Not Used")
 
+# Formatting for different filters in the vcf file
+VARIANT_FILTERS = {
+    "pass": {"label": "PASS", "description": "Passed filtering", "label_class": "success"},
+    "germline": {"label": "GERM", "description": "Germline variant", "label_class": "warning"},
+    "germline_risk": {"label": "GERM", "description": "Germline risk", "label_class": "warning"},
```

Comment:
These are filtered out in BALSAMIC. We don't report any germline or germline_risk in BALSAMIC.